<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerOrderPage</class>
 <widget class="QDialog" name="CustomerOrderPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid;</string>
       </property>
       <property name="text">
        <string>Customer Order</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="orderForm" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>350</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#orderForm{ border: 1px solid; }</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="pickupForm">
          <item>
           <widget class="QLabel" name="pickupLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Delivery Locations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="DeliveryComboBox">
            <item>
             <property name="text">
              <string>London to Edinburgh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nottingham to Coventry</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Leicester to Cardiff</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dimentionsForm_4">
          <item>
           <widget class="QLabel" name="dimentionsLabel_3">
            <property name="text">
             <string>Length:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lengthField">
            <property name="placeholderText">
             <string>Less than 5 meters</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dimentionsForm_2">
          <item>
           <widget class="QLabel" name="dimentionsLabel_2">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="widthField">
            <property name="placeholderText">
             <string>Less than 2.5 meters</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dimentionsForm">
          <item>
           <widget class="QLabel" name="dimentionsLabel">
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="heightField">
            <property name="placeholderText">
             <string>Less than 2.4 meters</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="weightForm">
          <item>
           <widget class="QLabel" name="weightLabel">
            <property name="text">
             <string>Weight:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="weightField">
            <property name="placeholderText">
             <string>Less than 2000 kg</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Conditions:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ConditionComboBox">
            <item>
             <property name="text">
              <string>No condition</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Frozen</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fragile</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hazardous</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Perishable</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="weightForm_5">
          <item>
           <widget class="QLabel" name="weightLabel_5">
            <property name="text">
             <string>Item Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="itemNameField">
            <property name="placeholderText">
             <string>Required</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="verticalWidget" native="true">
       <property name="styleSheet">
        <string notr="true">#verticalLayout_3{ border: 1px solid; }</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="dimentionsLabel_4">
          <property name="styleSheet">
           <string notr="true">#dimentionsLabel_4{ border: 1px solid; }</string>
          </property>
          <property name="text">
           <string>Shipping Rate Calculation:

Calculate your order's shipping rate
after you insert the values on the left

Do not submit if you wish to see your shipping rate

Shipping rate calculations do not have proper validation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="shippingRateField">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Result</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calculateButton">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
